### ISSUE TITLE
Docker Desktop crashes while starting after installation

### ISSUE THREAD

**Issue by zunicd** [docker/for-win#3402]
- [x] I have tried with the latest version of my channel (Stable or Edge)
  - [x] I have uploaded Diagnostics
  - Diagnostics ID: 5FD94AC6-F2F5-4BA6-BB1B-05CC8FA3C5D2/20190216112402

### Expected behavior
Docker Desktop running
### Actual behavior
Docker Desktop crashed while starting after installation with the following error in the pop-up window:

Unable to create: The running command stopped because the preference variable "ErrorActionPreference" or common parameter is set to Stop: Failed to create a new virtual machine.

'MobyLinuxVM' failed to realize. (Virtual machine ID C30E5CBB-B72A-430F-9DF2-76BD2547BE04)

Failed to access configuration store: The system cannot find the path specified. (0x80070003).
at New-MobyLinuxVM, <No file>: line 191
at <ScriptBlock>, <No file>: line 411
   at Docker.Core.Pipe.NamedPipeClient.Send(String action, Object[] parameters) in C:\workspaces\stable-18.09.x\src\github.com\docker\pinata\win\src\Docker.Core\pipe\NamedPipeClient.cs:line 36
   at Docker.Actions.DoStart(SynchronizationContext syncCtx, Boolean showWelcomeWindow, Boolean executeAfterStartCleanup) in C:\workspaces\stable-18.09.x\src\github.com\docker\pinata\win\src\Docker.Windows\Actions.cs:line 92
   at Docker.Actions.<>c__DisplayClass19_0.<Start>b__0() in C:\workspaces\stable-18.09.x\src\github.com\docker\pinata\win\src\Docker.Windows\Actions.cs:line 74
   at Docker.WPF.TaskQueue.<>c__DisplayClass19_0.<.ctor>b__1() in C:\workspaces\stable-18.09.x\src\github.com\docker\pinata\win\src\Docker.WPF\TaskQueue.cs:line 59

 
### Information

  - Windows Version:
OS: Windows 10 Pro
Edition: Professional
Id: 1809
Build: 17763
BuildLabName: 17763.1.amd64fre.rs5_release.180914-1434
  - Docker for Windows Version:
Version: 2.0.0.3 (31259)
Channel: stable
Sha1: 8858db33c8692b69de9987a5d672798d778735b2
Started on: 2019/02/16 11:22:50.243
Resources: C:\Program Files\Docker\Docker\Resources

[log.txt](https://github.com/docker/for-win/files/2872164/log.txt)

### Steps to reproduce the behavior

  1. Re-installed Docker, the same issue
  2. Re-enabled Hyper-V and reinstalled Docker - the same issue

**Comment by DoctorWho8** [comment]
Ooops! I have a similar issue. With mine I went ahead and upgraded. Docker wasn't available to the Powershell terminal. I reinstalled it. Same problem. I then went ahead and did the upgrade. Same problem. Then during installation process from a freshly downloaded one from main Docker site, stable, it gave me this error messageL:
The process cannot access the file 'C:\Program Files\Docker\Docker\resources\docker-for-win.iso' because it is being used by another process.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize)
   at LightweightInstaller.UnpackArtifactsStep.<DoAsync>d__31.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at LightweightInstaller.InstallWorkflow.<HandleD4WPackage>d__26.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at LightweightInstaller.InstallWorkflow.<ProcessAsync>d__23.MoveNext()

What is Moby trying to tell me here?
